tailwind.config.js

```diff
@@ -3,13 +3,11 @@ const kvConfig = require('@kiva/kv-components/tailwind.config.js');
 module.exports = {
 	mode: 'jit',
 	presets: [kvConfig],
-	purge: {
```

Comment:
This syntax previously "fixed" the purging... is the updated syntax you have specific to the new version?

Reply:
None of their examples are using this object with the `content` array now...

Reply:
I think we got this syntax from the PurgeCss docs: https://purgecss.com/configuration.html#configuration-file

Reply:
Yeah, I think it was actually the act of changing anything in the purge setting that caused Tailwind to pick up the new styles for the previous fix. Some people's workarounds to this bug were to add an empty directory to the list and save. Then remove it and save. To get Tailwind to regenerate all the classes.

Reply:
> None of their examples are using this object with the `content` array now...

https://tailwindcss.com/docs/optimizing-for-production#enabling-manually

@ryan-ludwig @mcstover I think we still need the content array... I'll have to test My previous fix was specifically fixing the issue where you are using a class in UI that is not used in kv-ui-elements repo